### ISSUE TITLE
Donations Block - One-Time Donation - Make account creation optional

### ISSUE THREAD

**Issue by rosepajaroja** [Automattic/wp-calypso#49497]
### What

https://www.loom.com/share/cfb2ae51fa494a6e8cc2f6a03d44df9a

Make WordPress.com account creation for **One-Time Donation** optional.

### Why

I understand this is required for Monthly/Yearly Donations to give people access to cancel the recurring payment. However, making it optional for people making one-time donations will make the transaction easier and faster. 

A user came on chat today about this feature and requesting to get that prompt removed. 

### How

![Alt text](https://d.pr/i/yQhgft/NzYhyr2HVN+)

OR

![Alt text](https://d.pr/i/Ymi9YP+)

**Additional context**
hc-27370760

**Comment by arunsathiya** [comment]
Requested on 3951777-zen as well.

**Comment by kristarella** [comment]
Yes please! We used this block for a one-time fundraiser event and it feels odd to require account creation.

**Comment by ccwalburn** [comment]
Similar request to allow Payments block to not require WordPress.com account for one-time payments.
4321888-zen

**Comment by inaikem** [comment]
@rosepajaroja, do you happen to have access to the screenshot you linked in the original feature request above? 

If so, can you please edit the submission and upload them directly? Thanks!

**Comment by inaikem** [comment]
Adding this feature request for the Shilling Team project per this previous discussion: pNPgK-5vM-p2#comment-22328.

**Comment by michaeldcain** [comment]
While it's technically true that a user could donate without needing the payment management features that come with a WPcom account, this change would require a substantial refactor of how our Payments Blocks work on the backend and thus won't be prioritized by Shilling.

That being said, @Automattic/gold is working on a new version of Payments Block right now, and we can flag this for their work. /cc @eoigal
